pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
     "ansys-mapdl-reader>=0.51.7",
     "ansys-math-core>=0.1.2",
     "ansys-platform-instancemanagement~=1.0",
+    "ansys-tools-visualization-interface>=0.2.6<0.3.0",
```

Comment:
I'm used to have upper limits in dependencies to avoid possible breaking changes on update. I'll remove it since I see that PyMAPDL doesn't follow this approach